schema/adminschema/v1.json

```diff
@@ -44,7 +44,26 @@
                   "type": "boolean"
               }
           }
-      }
+      },
+      "ClientCreateRequest": {
+          "id": "ClientCreateRequest",
+          "type": "object",
+	      "description": "'client' field is a client registration request as defined by the OpenID Connect dynamic registration spec, and holds fields such as redirect URLs, prefered algorithms, etc. For brevity field names and types of that object have been omitted.",
+          "properties": {
```

Comment:
Nope, and that wouldn't be a great API either, taking an `interface{}` when always expects a certain struct.

 Maybe this API doesn't have to have the full OIDC dynamic reg stuff. Or maybe we reimplement it all in that JSON. 

The main thing is that the generated client code should work.